.github/workflows/pr.yml

```diff
@@ -1,10 +1,7 @@
 # apply cusom action when PR has been submitted
```

Comment:
While the changes made in the file are correct and functional, here are a few improvements that could help to maintain best practices:

1. It is recommended to add comments for each line describing its purpose in a workflow file. Including a detailed comment about what a job does can be really helpful for others to understand.
   
2. A good practice is to provide a name for the job "build" which would make it easier to find in the workflow run web interface.

3. For the 'on' keyword, it is better to keep the events related to the 'push' command as well so that the workflow gets triggered on both push and pull request events. 

Here is an optimized version of the file:

```yaml
# This workflow gets triggered when a pull request is made
on:
  # Workflow will run on any push to the main branch
  push:
    branches:
      - main
  
  # Workflow will run when a pull request gets opened, edited, synchronized or marked ready for review
  pull_request:
    types: [opened, edited, synchronize, ready_for_review]
   
jobs:
  # Job name 'Build Project' which is shown on the UI
  build:
    name: 'Build Project'
```